backend/Step_Time_Calculation.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+def calculate_step_time(force_data, threshold=20.0):
+    """
+    Calculate step time from force data collected from a Bertec treadmill.
+
+    Parameters:
+    force_data (list): A list of tuples where each tuple contains (time, force).
+    threshold (float): The force threshold to determine heel strikes and toe-offs.
+
+    Returns:
+    list: A list of calculated step times in seconds and moving averages.
+    """
+    step_times = []
+    step_start_time = None
+
+    for time, force in force_data:
+        # Detect heel strike
+        if force > threshold and step_start_time is None:
```

Comment:
I think this should be >= since if force == 20 (aka. 20N) then it would be considered a step.